docs/extensibility/payment-method-integration.md

```diff
@@ -0,0 +1,131 @@
+# Payment Method Integration for the Checkout Block
```

Comment:
There is a VS code extension I have been using which automatically generates a TOC. Let me know if you want the details, it's very good.

Reply:
nice!